### PR TITLE
final commit with the change in all the files done on local repository

### PR DESCRIPTION
**Pull request by 11chankun**
This is my final commit with all the local changes.

### CODE REVIEW

File: intents/main.py [hunk 1]
```diff
@@ -18,10 +18,10 @@
 It returns the result of the intent and also the list of suggestions.
 """
 
-import json, pandas, enum, topk, slice_compare
+import json, pandas, enum, topk, slice_compare, correlation, trend, time_compare
 from flask import escape
```
Comment (bhagyakjain) [inline intents/main.py:22]: Correlation Is not yet submitted to the Googleinterns:master . This change should be done after I submit correlation.py

File: intents/main.py [hunk 2]
```diff
@@ -18,10 +18,10 @@
...
 from show import show
-from util import enums
+from util import enums, insert_as_column
```
Comment (bhagyakjain) [inline intents/main.py:25]: Same with insert_as_column.py

File: intents/main.py [hunk 3]
```diff
@@ -85,40 +97,143 @@ def hello_http(request):
...
         query_result = topk.topk(query_table_dataframe,
                                  metric, dimensions, is_asc, k,
                                  summary_operator=summary_operator,
+                                 slices=slices_list,
                                  date_column_name=date_column_name,
-                                 date_range=date_range,
-                                 slices=slices_list
+                                 day_first=day_first,
+                                 date_range=date_range
                                 )
```
Comment (bhagyakjain) [inline intents/main.py:157]: date_range  using  day_first  is not yet submitted

File: intents/slice_compare.py [hunk 4]
```diff
@@ -43,15 +46,15 @@ def slice_compare(table, metric, dimensions, all_dimension, all_metric,
...
         all_metric: Type-list of str
             It is the list of metric columns in the initial table
         date_range: Type-tuple
             Tuple of start_date and end_date
         date_column_name: Type-str
             It is the name of column which contains date
-        date_format: Type-str
+        day_first: Type-str
             It is required by datetime.strp_time to parse the date in the format
             Format Codes
```
Comment (bhagyakjain) [inline intents/slice_compare.py:59]: Type-Bool

**Comment by bhagyakjain** [comment]
Summarizing the current changes this PR has:

1. Added Some tests
2. Updated expected_results in some tests
3. Wired the oversights in time_compare.py
4. Updated the main.py with the main.py on GCP

1, 2, 3 seems good, just that day_first should not be used until it is updated in the aspects (And with that change I would also make changes in all the intents as well)

4 - we should not merge this as there are many files it uses that are not yet submitted

**Comment by arijitde-goog** [comment]
Thanks Bhagya! Chandan, can you please remove (4) from the pull request?

**Pull request closed without merging**